### ISSUE TITLE
License Change: MIT to Apache 2.0

### ISSUE THREAD

**Issue by amye** [brigadecore/brigade#957]
Opening this issue to track license change from MIT to Apache 2.0 as part of completing onboarding into CNCF + LF. 
cc: @technosophos, @caniszczyk

**Comment by technosophos** [comment]
I think we want to change all of the code repos under `brigadecore/`, correct? And to my knowledge, all we have to do to switch off the MIT is check in a new `LICENSE`. No legal review needed, right?

**Comment by caniszczyk** [comment]
Yes, do a PR with the license change and get sign off on the maintainers. This should have been done as part of the transfer to CNCF when we dropped the CLA and move to ALv2+DCO

**Comment by krancour** [comment]
> all we have to do to switch off the MIT is check in a new LICENSE

Noticed yesterday that the footer on Kashti explicitly makes note of Brigade and Kashti both being licensed under the MIT license. So let's not forget to also change that.

**Comment by radu-matei** [comment]
All repositories now have Apache-2.0 license.
Closing.
